src/dsp_tools/commands/excel2json/resources.py

```diff
@@ -217,6 +212,23 @@ def excel2resources(
     return resources, True
 
 
+def _prepare_classes_df(resource_dfs: dict[str, pd.DataFrame]) -> tuple[pd.DataFrame, dict[str, pd.DataFrame]]:
+    if "classes" not in resource_dfs.keys():
```

Comment:
There is one question for me. Is it sufficient to allow uppercase and lower case or should we support other spellings (that will be more difficult).